__tests__/api-test.test.js

```diff
@@ -447,3 +449,91 @@ describe("/api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?queries", () => {
+  test("GET: 200, articles are filtered by topics when passed a topic query", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+        expect(articles.length).toBe(1);
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("topic", "cats");
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(Number));
+          expect(article).not.toHaveProperty("body");
+        });
+      });
+  });
+  test("GET: 200, article is ordered by another column thats not created_at", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("author", { descending: true });
+      });
+  });
+
+  test("GET: 200, article is passed two queries", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("title");
+      });  
+  });
+
+   test("GET: 200, article is passed three queries", () => {
+     return request(app)
+       .get("/api/articles?sort_by=votes&order=asc&topic=mitch")
+       .expect(200)
+       .then(({ body }) => {
+         const { articles } = body;
+         expect(articles).toBeSortedBy("votes");
+         expect(articles.length).toBe(12);
+         articles.forEach((article) => expect(article.topic).toBe("mitch"));
+       });
+   });
+  
+  test("GET: 200, article is passed article_id query", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toBeSortedBy("article_id");
+        expect(articles.length).toBe(13);
+      });
+  });
```

Comment:
Please add a test for a successful response where a topic doesn't have any articles.